lib/rack_graphql/application.rb

```diff
@@ -8,18 +9,30 @@ def self.call(
       log_exception_backtrace: RackGraphql.log_exception_backtrace,
       health_route: true,
       health_response_builder: RackGraphql::HealthResponseBuilder,
-      error_status_code_map: {}
+      error_status_code_map: {},
+      error_response_middleware: RackGraphql::ExceptionJSONResponseMiddleware,
+      middlewares: []
     )
+      # rubocop:enable Metrics/ParameterLists
 
       ::Rack::Builder.new do
+        if error_response_middleware
```

Comment:
since `error_response_middleware` is an argument and can be provided by the user, we except from the custom middleware to accept specific arguments (app_name and so on), right ?